client/src/App.js

```diff
@@ -1,13 +1,37 @@
 import "./App.css";
+import React, { useState } from "react";
+import { videosData } from "./Data";
+import Video from "./Video";
+import AddVideo from "./AddVideo";
+
+const App = () => {
+  const [videos, setVideos] = useState(videosData);
+
+  const handleUpVote = (video) => {};
+
+  const handleDownVote = (video) => {};
+
+  const handleRemoveVideo = (video) => {};
+
+  const handleAddVideo = (newVideo) => {};
```

Comment:
Don't you need to implement those functions for level 100?

Reply:
Sorry, my fault, will do that (